geometry_msgs/Pose pose_a
geometry_msgs/Pose pose_b
---
bool success
string message
